### DISCUSSION TITLE
Migrate from Play Store to F-Droid

### DISCUSSION THREAD

**Discussion by mal-tee** [streetcomplete/StreetComplete#2458] [Q&A] [answered]
Hi!

Great App! 

Currently I have SC installed via the Play Store. I would like to get it from F-Droid instead. Is there any way to retain my stats and achievements? Or are they saved in my OSM account anyway and I don't have to worry about them? :D

**Comment by matkoniecz** [comment, marked as answer]
Stats and achievements are synchronized, so if you install from F-Droid it will work fine (there may be a bit lag before they will appear after login).

This is stored as changeset tags, and synchronization is handled by server setup by @westnordost 

Note that F-Droid has some benefits, but in case of SC it is AFAIK not modified at all and suffers from delay in releases (as F-Droid build servers sometimes fail).

**Reply by mal-tee** [reply]
Thank you!
